### PR TITLE
Update url in setup.py

### LINKED ISSUE #71: The setup.py `url` is broken
Currently it points to "http://www.zope.org/Products/ZODB", which is a 404.

Most zopefoundation projects use their github page as the url these days, which has the nice side-effect of putting some extra information on PyPI.

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #71

Also fix the lint warnings in setup.py: consistent use of continued indentation and trailing commas, no spaces around = in keyword arguments

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-version = '4.2.5.dev0'
+version = '4.3.0.dev0'
```
Comment (tseaver) [inline setup.py]: Rebase to pick up that this is done already in #70?

Comment (jamadden) [reply setup.py]: I missed changing setup.py in that PR, I just hit CHANGES.rst.

Comment (tseaver) [reply setup.py]: That's why I usually avoid doing either of those changes until just before making the release.

**Comment by jamadden** [comment]
Thank you. I intend to release to PyPI now if there are no objections.

**Pull request merged**